test/browser/fixtures/server-url/vitest.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'vitest/config'
+import basicSsl from '@vitejs/plugin-basic-ssl'
+
+// test https by
+//   TEST_HTTPS=1 pnpm test-fixtures --root fixtures/server-url
+
+const provider = process.env.PROVIDER || 'webdriverio';
+const browser = process.env.BROWSER || (provider === 'playwright' ? 'chromium' : 'chrome');
+
+export default defineConfig({
+  plugins: [
+    // @ts-ignore basicSsl type error?
```

Comment:
This is such a react mindset 😄 `plugins` cannot have an empty string or undefined which can be produced with `&&` operator

Reply:
Ouch, that's bad too, but actually I'm getting a different error too probably due to old module resolution:

![image](https://github.com/vitest-dev/vitest/assets/4232207/bca4777d-7cbe-4662-b550-70c91fd15f12)

I think using `Bundler` is fine? https://github.com/vitest-dev/vitest/pull/4855/commits/e49f71c1952d23c69f130bb14e6e1a3d8c371ebf